recipe_scrapers/valdemarsro.py

```diff
@@ -0,0 +1,100 @@
+from ._abstract import AbstractScraper
+from ._utils import normalize_string
+from ._exceptions import ElementNotFoundInHtml
+
+import re
+
+
+class Valdemarsro(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "valdemarsro.dk"
+
+    def title(self):
+        return self.soup.h1.get_text().strip()
+
+    def category(self):
+        category_group_element = self.soup.find("div", {"class": "recipe-bar"})
+
+        categories = [
+            a.get_text()
+            for a in category_group_element.find_all(
+                lambda a: "/tag/" not in a.attrs["href"]
```

Comment:
Kinda nitpicking at small details here (this isn't a blocker), but out of curiosity: can you translate what the website is thinking in terms of categories here?  (I see the link elements, some of them with `/tag/` in the path -- like flour-related ones -- but I don't completely understand the arrangement of them)

Reply:
By all means, go ahead, I'm all for nitpicking 😅 

![image](https://user-images.githubusercontent.com/4853317/154158079-4fed8201-7705-4367-98a7-92259bcb7608.png)

Under each recipe, you'll find a number of clickable tags from which I've scraped the categories.

The tags that are not highlighted translate to:

* Campfire food
* Family favourites
* Food pancakes and food waffles
* Easy every day food
* Recipes
* Recipes for children - every day favourites children
* Pancakes
* Dinner side dishes
* Vegetarian

The tags that are highlighted, which are the ones (luckily) containing `/tag/` in the URL, translate to:

* Wheat flour
* Graham flour

I decided that while the former are actual categories, the latter are basically just an arbitrary subset of the ingredients.  
I figured that if you want to filter recipes by their ingredients, it would make sense to just base that on the ingredients list.

Reply:
Cool, thank you - much appreciated :)

Is the presence of flour-related categories in any way a cultural thing of trying to indicate recipes that might be gluten-free, or is it just coincidence?  (I agree that those two seem oddly out-of-context relative to the others)

Reply:
I think it's just a random coincidence that any and all ingredient tags were flour-related. I tried checking a random dinner recipe ([Viking Pot](https://www.valdemarsro.dk/vikingegryde/)), and found that out of 18 ingredients, the recipe was tagged with 15 of them.

The only ones not included are bacon(!!!), onions and rice, while paprika has no less than two tags (smoked and sweet, respectively). Looking for [recipes tagged with bacon](https://www.valdemarsro.dk/tag/bacon/), however, turns up lots of results, so it's pretty inconsistent.

I think the tags are applied to the recipes manually, rather than being derived from the ingredient list.

Reply:
Hehe, okie doke - thanks for explanation!